Lib/opcode.py

```diff
@@ -189,7 +189,8 @@ def pseudo_op(name, op, real_ops):
 def_op('DELETE_DEREF', 139)
 hasfree.append(139)
 jrel_op('JUMP_BACKWARD', 140)    # Number of words to skip (backwards)
-
+def_op('COMPARE_AND_BRANCH', 141)   # Comparison and jump
+hascompare.append(141)
```

Comment:
This opcode needs to be in hasjrel as well.

Reply:
The problem is that `dis` sees `COMPARE_AND_BRANCH`, then `POP_JUMP_IF_`. The `POP_JUMP_IF_` is where we jump from.
So marking `COMPARE_AND_BRANCH` as a jump confuses dis, it thinks the operator is a jump offset. `COMPARE_AND_BRANCH` is effectively a superinstruction.

I could special case `COMPARE_AND_BRANCH` in dis, but I think it might be better to wait for proper support for longer instructions.